lib/voicebase/client.rb

```diff
@@ -26,6 +26,10 @@ def initialize(args = {})
       @user_agent          = args[:user_agent] || "usertesting-client/#{VoiceBase::version}"
       @locale              = args[:locale] || 'en'  # US English
 
+      if ENV['VOICEBASE_BEARER_TOKEN']
+        @token = VoiceBase::Client::Token.new(ENV['VOICEBASE_BEARER_TOKEN'])
+      end
```

Comment:
what happens if @token is not set?

Reply:
Previously, that was the standard case - the code in `lib/voicebase/v2/client.rb` authenticates and sets the token if it's not set when making a query. In the normal flow until now, the token gets  acquired by requesting it from voicebase when the client makes its first request.

Reply:
[Here](https://github.com/usertesting/voicebase-client-ruby/blob/master/lib/voicebase/client.rb#L29), the client gets a different module included based on the api version. Then if you look [here](https://github.com/usertesting/voicebase-client-ruby/blob/master/lib/voicebase/v2/client.rb#L149) in the v2 client that we actually use, you can see the conditional that authenticates (setting `token`) if `token` is not yet set.